index.html

```diff
@@ -38,29 +38,226 @@ <h1>
+          <li class="tech-tag-name"><span class="tech-tag-text">html</span></li>
+          <li class="tech-tag-name"><span class="tech-tag-text">css</span></li>
+          <li class="tech-tag-name">
+            <span class="tech-tag-text">javascript</span>
+          </li>
+        </ul>
+        <div class="btn-container">
+          <button type="button" class="view-project-btn">See Project</button>
+        </div>
+      </article>
+      <article class="portfolio-item">
+        <img src="./assets/images/tonic2.png" alt="tonic" class="mt-12" />
+        <div class="title-block mt-12">
+          <h2 class="project-title">Tonic</h2>
+          <div class="project-role">
+            <p class="client">CANOPY</p>
+            <img src="./assets/Icons/Counter.svg" alt="seperator" />
+            <p class="role">Back End Dev</p>
+            <img src="./assets/Icons/Counter.svg" alt="seperator" />
+            <p class="year">2015</p>
+          </div>
+        </div>
+        <p class="description-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tech-tags">
+          <li class="tech-tag-name"><span class="tech-tag-text">html</span></li>
+          <li class="tech-tag-name"><span class="tech-tag-text">css</span></li>
+          <li class="tech-tag-name">
+            <span class="tech-tag-text">javascript</span>
+          </li>
+        </ul>
+        <div class="btn-container">
+          <button type="button" class="view-project-btn">See Project</button>
+        </div>
+      </article>
+      <article class="portfolio-item">
+        <img src="./assets/images/tonic.png" alt="tonic" class="mt-12" />
+        <div class="title-block mt-12">
+          <h2 class="project-title">Tonic</h2>
+          <div class="project-role">
+            <p class="client">Multi-Post Stories</p>
+            <img src="./assets/Icons/Counter.svg" alt="seperator" />
+            <p class="role">Back End Dev</p>
+            <img src="./assets/Icons/Counter.svg" alt="seperator" />
+            <p class="year">2015</p>
+          </div>
+        </div>
+        <p class="description-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tech-tags">
+          <li class="tech-tag-name"><span class="tech-tag-text">html</span></li>
+          <li class="tech-tag-name"><span class="tech-tag-text">css</span></li>
+          <li class="tech-tag-name">
+            <span class="tech-tag-text">javascript</span>
+          </li>
+        </ul>
+        <div class="btn-container">
+          <button type="button" class="view-project-btn">See Project</button>
+        </div>
+      </article>
+    </section>
+
+    <section class="about-section">
+      <div class="about-top">
+        <h2 class="about-primary-text">About<br />Myself</h2>
+        <p class="about-secondary-text">
```

Comment:
- [ ] Good job on adding both sections but kindly fix this issue in the middle of them. It doesn't need to have this much space following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Here you have a screenshot of how it is looking on my end:
![image](https://user-images.githubusercontent.com/74506933/153711207-03729b9c-e3d3-4268-9cf4-48773093dbb5.png)